packages/e2e-test-utils-playwright/src/request-utils/index.ts

```diff
@@ -108,25 +115,29 @@ class RequestUtils {
 		this.baseURL = baseURL;
 	}
 
-	login = login;
-	setupRest = setupRest;
-	rest = rest;
-	getMaxBatchSize = getMaxBatchSize;
-	batchRest = batchRest;
-	getPluginsMap = getPluginsMap;
-	activatePlugin = activatePlugin;
-	deactivatePlugin = deactivatePlugin;
-	activateTheme = activateTheme;
+	login = login.bind( this );
+	setupRest = setupRest.bind( this );
+	rest = rest.bind( this ) as < RestResponse = any >(
+		options: RestOptions
+	) => Promise< RestResponse >;
```

Comment:
The reason for using `as` here is that the type argument would be lost if `bind` is used.

Is there a better way?  @kevin940726

Reply:
Good question, I don't know a good solution!

The most elegant solution I found is:

```suggestion
	// .bind() drops the generic types. Re-casting it to keep the type signature.
	rest = rest.bind( this ) as typeof rest;
```